full-service/src/json_rpc/v1/api/wallet.rs

```diff
@@ -877,9 +891,9 @@ where
             }
         }
         JsonCommandRequest::get_txo { txo_id } => {
-            let (txo, status, _) = service.get_txo(&TxoID(txo_id)).map_err(format_error)?;
+            let txo_info = service.get_txo(&TxoID(txo_id)).map_err(format_error)?;
             JsonCommandResponse::get_txo {
-                txo: Txo::new(&txo, &status),
+                txo: Txo::new(&txo_info.txo, &txo_info.status),
```

Comment:
Might be cleaner to `impl From<&TxoInfo> for Txo` so that you can just do `Txo::from(&txo_info)` instead of having to repeat this